### ISSUE TITLE
PhpOffice\PhpWord\Element\AbstractContainer error suppression

### ISSUE THREAD

**Issue by daum** [PHPOffice/PHPWord#468]
In AbstractContainer.php on line 101 right now there is the following:
```php
@list($count, $fontStyle, $paragraphStyle) = $args; // Suppress error
```

The issue is if a user has listeners for warnings/errors that they will still get the error which then can cause their scripts to send extra alert emails.  Switching this to the following would have the same affect, just never produce the warnings to the listeners:

```php
if (isset($args[0])) {
    $count = $args[0];
}
else {
    $count = null;
}
// Same for rest of variables
```

I'll submit a PR with the full fix.  While this does have a few more lines of code, it will allow error monitoring to continue without false positives.

**Comment by GMTA** [comment]
> The issue is if a user has listeners for warnings/errors that they will still get the error which then can cause their scripts to send extra alert emails.

Yeah, if you use [set_error_handler()](http://php.net/set_error_handler) then you should also read the remark about it bypassing [error_reporting()](http://php.net/manual/en/function.error-reporting.php) completely. This is not PHPWord at fault with error suppression, this is your custom error handling ignoring the error level.

Fix your code:
```PHP
if (!(error_reporting() & $errno)) {
    return;
}
```
